conanfile.py

```diff
@@ -23,15 +23,19 @@ def requirements(self):
         if self.options.asio == 'standalone':
             self.requires.add('asio/1.13.0')
         else:
-            # 1.70 doesn't work: https://github.com/zaphoyd/websocketpp/issues/794
-            self.requires.add('boost/1.69.0@conan/stable')
+            self.requires.add('boost/1.71.0')
 
     def source(self):
         archive_name = "{0}-{1}".format(self.name, self.version)
         tools.get("{0}/archive/{1}.tar.gz".format(self.homepage, self.version),
                   sha256="178899de48c02853b55b1ea8681599641cedcdfce59e56beaff3dd0874bf0286")
         os.rename(archive_name, self._source_subfolder)
 
+        # Patch for boost 1.70+ support
```

Comment:
Could you please directly mention here, that this patch won't be necessary anymore with the release after 0.8.1?

Reply:
Done